tests/functional/settings_devices.js

```diff
@@ -132,7 +132,7 @@ define([
         .findByCssSelector('.device:nth-child(1) .device-name')
           .getVisibleText()
           .then(function (val) {
-            assert.equal(val, BROWSER_DEVICE_NAME + ' (current)', 'device name is correct');
+            assert.equal(val, BROWSER_DEVICE_NAME, 'device name is correct');
```

Comment:
Can you replace line 132-137 with a call to [testElementTextEquals](https://github.com/mozilla/fxa-content-server/blob/0ff2e355efb898c82822dc5bbfbdab9351f25b21/tests/functional/lib/helpers.js#L1034) as well?